docs/podman-create.1.md

```diff
@@ -508,6 +508,14 @@ By default a container will have its root filesystem writable allowing processes
 to write files anywhere.  By specifying the `--read-only` flag the container will have
 its root filesystem mounted as read only prohibiting any writes.
 
+**--restart=""**
+
+Not implemented.
+
+Restart should be handled via a systemd unit files. Please add your podman
+commands to a unit file and allow systemd or your init system to handle the
+restarting of the container processes.
```

Comment:
Is there anywhere else we can point them to for more info?  systemd(1) or something?

Reply:
Added example and link to systemd.unit man page.